pkg/virt-controller/watch/vmi_test.go

```diff
@@ -3189,7 +3189,7 @@ var _ = Describe("VirtualMachineInstance watcher", func() {
 		Context("hotplug operation", func() {
 			BeforeEach(func() {
 				vmi = api.NewMinimalVMI(vmName)
-				pod = NewPodForVirtualMachine(vmi, k8sv1.PodRunning)
+				pod = NewPodForVirtualMachine(vmi, k8sv1.PodRunning, networkv1.NetworkStatus{})
```

Comment:
Adding a single  `networkv1.NetworkStatus` object will make the test pod have Multus network-status annotation.
Without it, the test fails with "network -status annotation is missing.."